quiz.js

```diff
@@ -0,0 +1,25 @@
+var treeObject = {};
+document.getElementById("growTreeBtn").addEventListener("click", tree);
+document.getElementById("character").addEventListener("keydown", enterListener);
+document.getElementById("treeHeight").addEventListener("keydown", enterListener);
+
+function enterListener(key) {
+	if (13 == key.keyCode) {
+		tree()
+	}
+}
+
+
+function tree() {
```

Comment:
**Requirement not met:** Create a tree function that should build a pine tree out of a character in the Chrome dev tools console. It accepts a single object as an argument.